src/app/drivingLicence/controllers/root.js

```diff
@@ -1,20 +1,74 @@
 const { Controller: BaseController } = require("hmpo-form-wizard");
 
+const {
+  API: {
+    BASE_URL,
+    PATHS: { PERSON_INFO }
+  }
+} = require("../../../lib/config");
+
 class RootController extends BaseController {
   async saveValues(req, res, next) {
-    const sharedClaims = req.session?.shared_claims;
+    req.sessionModel.reset();
+    req.sessionModel.set("isAuthSourceRoute", false);
 
-    if (sharedClaims) {
-      if (sharedClaims?.names?.length > 0) {
-        req.journeyModel.set("surname", sharedClaims.names[0]?.familyName);
-        req.journeyModel.set("givenNames", sharedClaims.names[0]?.givenNames);
-      }
+    const headers = {
+      session_id: req.session?.tokenId
+    };
+
+    //axios.post to new personInfo endpoint, put licence details into req.session.shared_claims shared claims
+    const personInfoApiResponse = await req.axios.post(
```

Comment:
For security can we wrap this in an if statement with an env var so we can merge without triggering new code yet

Reply:
Yes this actually means I can remove feature toggle from corresponding API PR